istats: add page (#2969)
tldr-pages/tldr: pages/osx/istats.md

# istats

> > CLI tool that shows statistic like CPU temperature, fan speeds and battery informations.

Reviewer: Perhaps this will read better?
```suggestion
> CLI tool that shows statistics such as CPU temperature, fan speeds and battery status.
```
